modules/apps/calendar/calendar-web/src/main/resources/META-INF/resources/css/main.scss

```diff
@@ -217,6 +217,7 @@
 	.scheduler-event-content,
 	.scheduler-event-name,
 	.scheduler-event-title {
+		color: #272833;
```

Comment:
@larissacribeiro You should be using the variable `$gray-900` instead of the hexadecimal directly. Reference: https://clayui.com/docs/css/color.html#clay-sidebar.

Reply:
Just did it!